configuration/pih/htmlforms/dispensing.xml

```diff
@@ -286,39 +286,39 @@
                     <obs id="frequencyCoded{0}" class="frequency select-arrow" conceptId="CIEL:160855"
                          answerConceptIds=
                                  "PIH:OD,PIH:BID,PIH:TID,PIH:QID,PIH:5D,PIH:6D,PIH:7D,PIH:8D,PIH:9D,PIH:OM,PIH:ON,PIH:PRN,PIH:STAT,PIH:Q2H,PIH:Q3H,PIH:Q4H,PIH:Q6H,PIH:Q12H,PIH:5622"
-                         answerCodes="OD,BID,TID,QID,5D,6D,7D,8D,9D,OM,ON,SOS,STAT,Q2H,Q3H,Q4H,Q6H,Q12H,dispensing.general.other" />
+                         answerCodes="OD,BID,TID,QD,5D,6D,7D,8D,9D,OM,ON,SOS,STAT,Q2H,Q3H,Q4H,Q6H,Q12H,dispensing.general.other" />
                </p>
 
                 <p class="inline">
                     <label>
                         <uimessage code="mirebalais.dispensing.medicationDuration"  />
                     </label>
                     <obs id="duration{0}" class="duration doseInput" conceptId="CIEL:159368"/>
-                        <obs id="durationUnit{0}" class="duration-unit select-arrow" conceptId="PIH:6412" answerConceptIds="PIH:Days,PIH:1073,PIH:Months,PIH:Hours"/>
+                        <obs id="durationUnit{0}" class="duration-unit select-arrow" conceptId="PIH:6412" answerConceptIds="PIH:Days,PIH:1073,PIH:Months,PIH:Hours,CIEL:1733"/>
                 </p>
                 <p class="inline">
                     <label>
                         <uimessage code="pihcore.drug.route"/>
                     </label>
                     <obs id="route-{0}" class="route select-arrow" conceptId="PIH:12651"
-                         answerConceptIds="CIEL:160240,CIEL:160242,CIEL:160243,CIEL:160245"
-                         answerLabels="Oral,IV,IM,SubQ"/>
+                         answerConceptIds="CIEL:160240,CIEL:160242,CIEL:160243,CIEL:160245,CIEL:162797"
+                         answerLabels="Oral,IV,IM,SubQ,Apply"/>
```

Comment:
I don't like "apply" because you can apply to an eye, ear, tongue, skin, etc.  You like want this to show that it is applied to the skin (tipical).  I think it's better for the EMR to say "topical".  But it's up to you.  You're using the right concept @tommyiversonj